atef/bin/config.py

```diff
@@ -18,6 +19,39 @@ def build_arg_parser(argparser=None):
     if argparser is None:
         argparser = argparse.ArgumentParser()
 
+    # Arguments that need to be passed through to Qt
+    qt_args = {
+        '--qmljsdebugger': 1,
+        '--reverse': '?',
+        '--stylesheet': 1,
+        '--widgetcount': '?',
+        '--platform': 1,
+        '--platformpluginpath': 1,
+        '--platformtheme': 1,
+        '--plugin': 1,
+        '--qwindowgeometry': 1,
+        '--qwindowicon': 1,
+        '--qwindowtitle': 1,
+        '--session': 1,
+        '--display': 1,
+        '--geometry': 1
+    }
+
+    for name in qt_args:
+        argparser.add_argument(
+            name,
+            type=str,
+            nargs=qt_args[name]
+        )
+
+    argparser.add_argument(
+        '--style',
+        type=str,
+        choices=QStyleFactory.keys(),
+        default='fusion' if sys.platform == 'darwin' else None,
```

Comment:
Is there any reason why we specifically wanted this only on osx? I know the ticket only mentions mac but maybe it's simpler to just always set it to fusion, but maybe there's some detail I don't know where this breaks something.

typhos just unilaterally applies this as the default (using a different method)
https://github.com/pcdshub/typhos/blob/c1a964aa29e67d35c1ed3ff7c58158a9325e642a/typhos/utils.py#L362

Reply:
A good point.  I don't think it was a fix we only wanted on osx, I interpreted that ticket as the only seeing the issue in osx.  Making the default fusion is probably fine here

Reply:
Sorry, didn't see this yesterday. If we do want to default to fusion, do we want the argument passthrough at all? Initially I was just going to set the style to fusion right after `QApplication` is constructed, but decided on the argument passthrough instead.

Reply:
I think having the option to set the style via cli args is valuable, we can just set the default to fusion regardless of OS